src/repos/player.js

```diff
@@ -0,0 +1,148 @@
+var sql = require('pg-sql').sql
```

Comment:
Kind of a general readability nit.  I find very vertically spread out SQL to be much harder to read.  Have you considered moving toward a slightly more dense style?

For example, I find
```
FROM
  player
JOIN
  steam_user
ON
  steam_user.steam_id = player.steam_id
JOIN
  season
ON
  season.id = player.season_id
WHERE
  1 = 1
```
much harder to grok than
```
FROM player
JOIN steam_user ON steam_user.steam_id  = player.steam_id
JOIN season on season.id = player.season_id
WHERE 1 = 1
```

I'm agree with splitting the very long selects by column because they can't fit on one line, but the rest fits quite comfortably.  Thoughts?